tests/Unit/ArtisanTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Tests\Unit;
+
+use LaraCrafts\UrlShortener\Console\Commands\ShortenCommand;
+use LaraCrafts\UrlShortener\UrlShortenerManager;
+use Orchestra\Testbench\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+class ArtisanTest extends TestCase
+{
+    /** @var \Symfony\Component\Console\Output\BufferedOutput */
+    protected $output;
+
+    /** @var \Mockery\MockInterface */
+    protected $shortener;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setUp() : void
+    {
+        parent::setUp();
+
+        $this->output = new BufferedOutput();
+        $this->shortener = \Mockery::mock(UrlShortenerManager::class);
+        $this->app['url.shortener'] = $this->shortener;
+    }
+
+    public function testShortenCommand()
+    {
+        $command = new ShortenCommand();
+
+        $command->setLaravel($this->app);
+
+        $url = str_random(20);
```

Comment:
Please replace this with the class function, this function is deprecated as of Laravel 5.8 (don’t forget to import the class if you do)

```suggestion
        $url = Str::random(20);
```